### DISCUSSION TITLE
part-ii/2020-136/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#51] [Question comments]
Part II 2020 Riemann Surfaces: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2020-136/

**Comment by artie2000** [comment]
<details>
<summary>For the last part use:</summary>

$\langle 1,\zeta_6\rangle$
</details>

**Comment by silkca** [comment]
The first part is all bookwork.

The second part is an immediate consequence of a standard result of half-lattice points.

For the final part, bear in mind there is a very nice way to visualise how conformal equivalences of complex tori act (which I believe is alluded to on an example sheet, but not explicitly stated at all, although I may be wrong): the action of $\theta$ on $\mathbb{C} / \Lambda$ can be lifted through the quotient map to act on $\mathbb{C}$, and this action is forced to be scalar multiplication. In this case, for $\theta$ to act transitively on the half-lattice points, the most natural thing to do is to try using a lattice which produces half-lattice points on the unit circle, and try multiplication by a well-chosen number on the unit circle.
